index.js

```diff
@@ -4,7 +4,7 @@
 const AWS = require('aws-sdk')
 const S3 = new AWS.S3({signatureVersion: 'v4'});
 const Sharp = require('sharp');
-const PathPattern = new RegExp("(.*/)?(.*)/(.*)");
+const PathPattern = new RegExp("(.*\/)?(.*)\/(.*)");
```

Comment:
It definitely must not be an issue, since `'\/' === '/'`.
Do you think it looks better?

Reply:
It is not a issue because `'/'` is not a special character and `RegExp` convert to `'\/'`, but in JavaScript  `\/ !== /`, a slash symbol is used to open and close the regexp, `/\//g`,  so i prefer escape it too.

Reply:
_"RegExp convert to `'\/'`"_
If you mean the class (and the constructor that we are calling), then no, it does not. RegExp class has nothing to do with that, in fact, it could not distinguish whether it were `'/'` or `'\/'`, since these strings are the same in JavaScript.

_"but in JavaScript `\/ !== /`"_
Could you please provide an example where it prints `true` for the input `'\/' !== '/'`?
Because as far as I am concerned, `'\/' === '/'` and `'\/'.length === 1`.
https://repl.it/repls/OrdinaryUntriedScope

---

#### However, we could change this into a literal expression!
You can see the literal expression `/(.*\/)?(.*)\/(.*)/`, and now we have to escape slashes.
To be honest, I cannot remember why I did not used that.
As we do not change the expression, I think it will be better simply to convert it into the literal one.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Creating_a_regular_expression

Reply:
Changed on my own.